### DISCUSSION TITLE
Adding Invite Flags?

### DISCUSSION THREAD

**Discussion by syncblaze** [hikari-py/hikari#2099] [Ideas]
I know that generally hikari does not implement undocumented things but hear me out:

Invites have "flags" you dont recieve them when fetching the invite but you can get them when listening to AuditLogEntryCreate Events, there when an Invite is Created or Deleted you actually get the flags field send in the AuditLogEntry.changes list. I think it would be nice to implement these flags although they are not listed in the official documentation since you can still encounter them when using hikari. More information about invite flags is documented here:

https://docs.discord.sex/resources/invite#invite-flags

I would be happy to create a PR implementing this if you guys think it is okay to implement that.

**Comment by syncblaze** [comment]
So i figured out that when you fetch an invite, flags are also sent, only when the flags are NONE they are not part of the response

**Comment by rayakame** [comment]
this has been documented and implemented in #2768
